README.md

```diff
@@ -1,8 +1,304 @@
-# RFC
+# Amethyst RFCs
+[Amethyst RFCs]: #amethyst-rfcs
 
-This is the repository where we work and keep track of the all RFC documents related to Amethyst.
-Having this repository allows people to participate in decision making by opening a pull request with a new document.
+- [Discussing RFCs](https://github.com/amethyst/rfcs/issues)
+- [Accepted RFCs](https://github.com/amethyst/rfcs/tree/master/accepted)
```

Comment:
All RFCs merged here are accepted, so I don't think we need an extra folder. I think we can just call that `rfcs/`. If you want to, we could group them (`engine`, `laminar`, `administrativa`).